init.lua

```diff
@@ -90,9 +75,10 @@ minetest.register_chatcommand("choosecase", {
 	func = function(name, params)
 		local lname = params:lower()
 		local worldpath = minetest.get_worldpath()
-		for iname, data in pairs(minetest.auth_table) do
+		local auth = minetest.get_auth_handler()
+		for iname in pairs(auth.iterate()) do
```

Comment:
Sauth doesn't implement the Auth handler API correctly in this instance, it should return an iterator not a table. You can create an iterator using `pairs()`

So, this needs to be changed to the line I said, and sauth needs to be fixed as I said above

Reply:
Ah okay thanks for pointing that out :+1: